test/unit/auth-handlers.test.js

```diff
@@ -1038,12 +1036,12 @@ describe('Auth Handler:', function () {
             handler.sign(authInterface, request, _.noop);
 
             // Original request should not have the timestamp and nonce
-            expect(_.get(rawRequests.hawk, 'auth.hawk.nonce')).to.not.be.ok();
-            expect(_.get(rawRequests.hawk, 'auth.hawk.timestamp')).to.not.be.ok();
+            expect(_.get(rawRequests.hawk, 'auth.hawk.nonce')).to.be.undefined;
```

Comment:
Are you checking for absence, or the value being explicitly `undefined`? Switch to the nested assertion helper if applicable. (Same goes for the assertions below)